### ISSUE TITLE
Cannot set 'alt' text of photo

### ISSUE THREAD

**Issue by ankon** [Collaborne/paper-drawer#2]
For accessibility it is important that we provide suitable alt-texts, and especially the photo in the paper-drawer-title should have one.

Currently there is no way for doing that.
